cli/src/cli-rpc-ops.c

```diff
@@ -5665,6 +5673,9 @@ cmd_profile_volume_brick_out(dict_t *dict, int count, int interval)
     cli_out("%12s: %" PRId64 " seconds", "Duration", sec);
     cli_out("%12s: %" PRId64 " bytes", "Data Read", r_count);
     cli_out("%12s: %" PRId64 " bytes", "Data Written", w_count);
+    if (interval >= 0) {
+        cli_out("%12s: %" PRId64, "FOP_HITS_PER_SEC", fop_hits_per_sec);
```

Comment:
For user-facing info, "FOP_HITS_PER_SEC" doesn't seem a user friendly name. Maybe just "fops/s" or something like that would be better.

Reply:
done